cg/models/scout/scout_load_config.py

```diff
@@ -78,28 +105,27 @@ class ScoutCancerIndividual(ScoutIndividual):
 class ScoutLoadConfig(BaseModel):
     owner: Annotated[str, BeforeValidator(field_not_none)] = None
     family: Annotated[str, BeforeValidator(field_not_none)] = None
-    family_name: str | None = None
-    synopsis: str | None = None
-    phenotype_terms: list[str] | None = None
-    phenotype_groups: list[str] | None = None
-    gene_panels: list[str] | None = None
+    family_name: Optional[str] = None
+    synopsis: Optional[str] = None
+    phenotype_terms: Optional[list[str]] = None
+    phenotype_groups: Optional[list[str]] = None
+    gene_panels: Optional[list[str]] = None
     default_gene_panels: list[str] = []
-    cohorts: list[str] | None = None
-    human_genome_build: str = None
-
-    rank_model_version: str | None = None
-    rank_score_threshold: int = None
-    sv_rank_model_version: str | None = None
-    analysis_date: datetime | None = None
+    cohorts: Optional[list[str]] = None
+    human_genome_build: Optional[str] = None
+    rank_model_version: Optional[str] = None
+    rank_score_threshold: Optional[int] = None
+    sv_rank_model_version: Optional[str] = None
+    analysis_date: Optional[datetime] = None
     samples: list[ScoutIndividual] = []
-    delivery_report: str
-    coverage_qc_report: str | None = None
-    cnv_report: str | None = None
-    multiqc: str | None = None
+    custom_images: CustomImages = CustomImages()
+    delivery_report: Optional[str] = None
+    coverage_qc_report: Optional[str] = None
+    cnv_report: Optional[str] = None
+    multiqc: Optional[str] = None
```

Comment:
Using `| None` is the more modern way. Optional[] was its predecessor.

Reply:
Reverted

Reply:
Oups, I have it backwards! Corrected